native/packages/react-native-bpk-component-banner-alert/src/BpkBannerAlert-test.ios.js

```diff
@@ -16,9 +16,10 @@
  * limitations under the License.
  */
 
+/* @flow */
+
 import commonTests from './BpkBannerAlert-test.common';
 
-jest.useFakeTimers();
```

Comment:
I have no idea what this was for, but the tests still pass without it.

Reply:
it was due to using `Date.now` but you're now spying on it so it should be safe to remove